pom.xml

```diff
@@ -97,6 +97,27 @@
             <version>${lombok.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.2.11</version>
```

Comment:
Please move all dependencies versions into properties section. It will be easier to maintain it in future.